### ISSUE TITLE
Export single note

### ISSUE THREAD

**Issue by cobblepot1** [melat0nin/zotero-roam-export#30]
Often, after exporting a source from Zotero to Roam, I want to add highlights or comments to the pdf and extract them with Zotfile, creating a new note in Zotero. To get that note into Roam, I have to export all item data and then delete duplicate information in Roam, which is a pain. If I manually copy/paste the note content, I lose the note links and styling. Instead, I would like Zotero-Roam-export to allow me to export an item note attachment to a JSON file that could import into Roam with the same Roam page name but not all of the duplicate data.

(I mentioned this in a closed note but no response so I wanted to make it visible)

**Comment by melat0nin** [comment]
To clarify, you want an export with only the item's title and the selected note's content?

Just to note also that in the above case the content of the new note will be appended to what's already on the Roam page, so you'd need to delete the previous note's blocks from the Roam page manually.

**Comment by cobblepot1** [comment]
Yes, that's what I want. I know that if I was exporting the same note again, I would need to manually delete duplicate blocks. Usually, I just want to add content from a new note. Thanks for considering this request!

I often have several separate notes on an item, so ideally if an item was highlighted, there would be an option 'Export item to Roam' and if a single note was highlighted it would say 'Export note to Roam'.
